packages/middleware-render-error-info/README.md

```diff
@@ -0,0 +1,4 @@
+
+## @dotcom-reliability-kit/middleware-render-error-info
+
+Express middleware to render error information in a way that makes debugging easier. This module is part of [FT.com Reliability Kit](https://github.com/Financial-Times/dotcom-reliability-kit#readme).
```

Comment:
```suggestion
Express middleware to render error information in a browser in a way that makes debugging easier. This module is part of [FT.com Reliability Kit](https://github.com/Financial-Times/dotcom-reliability-kit#readme).
```

To make explicit where the error information will be viewable (although 'render' does already heavily imply this).